libs/blocks/featured-article/featured-article.css

```diff
@@ -38,6 +38,10 @@
   padding-right: 0;
 }
 
+.featured-article-card-category {
```

Comment:
If this always had content, I imagine it wouldn't need the `15px` min height? Should this have a `:not(:empty)` appended so we don't get empty elements occupying space on the page? Or is that the intention?

Reply:
This is to avoid CLS. This container gets created. Then, the taxonomy look up happens. So, to avoid the little jump when the category and link are added I predefined the container with the height.